### DISCUSSION TITLE
不确定的行数请放在这里，因为 README 不适合放置可以清除的内容

### DISCUSSION THREAD

**Discussion by ahdg6** [Tartaros-Gamer/GriefDefenderSinicization#3] [General]
Not Sure Lines:

- 5 参考zh_CN(1.5.xx).conf，是否为所有的母领地？母领地是否为与子领地相对的上层领地？
- 7 参考zh_CN(1.5.xx).conf，是否为领地数而不是方块数？
- 11 货币银行是什么机制？

**Comment by pys078** [comment]
好的👌下次pr中将在README中删除
